### PR TITLE
Update how cloud domain and regions are used

### PR DESCRIPTION
**Pull request by nephomaniac**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 
-__version__ = '0.4.3.0.1'
+__version__ = '1.4.3.0.1.1'
 
 setup(name="adminapi",
       version=__version__,
       description="Eucalyptus Cloud Services and General System Administrative Utilities",
       long_description="Eucalyptus Cloud Services and General System Administrative Utilities",
-      url="https://github.com/bigschwan/adminapi",
+      url="https://github.com/nephomaniac/adminapi",
```
Comment (lincolnthomas) [inline setup.py:11]: Just curious, why these and not https://github.com/eucalyptus/nephoria ? (I'd read this PR to understand more about domains/regions and AWS vs Euca)

File: setup.py [hunk 2]
```diff
@@ -2,13 +2,13 @@
 
 from setuptools import setup, find_packages
 
-__version__ = '0.4.3.0.1'
+__version__ = '1.4.3.0.1.1'
```
Comment (lincolnthomas) [inline setup.py:5]: Love the deep versioning scheme! wow, 6 levels!?

File: setup.py (hunk 1 above)
Comment (nephomaniac) [reply setup.py:11]: Hi @lincolnthomas, 
I just updated the url per https://github.com/eucalyptus/adminapi/pull/36
Thanks!

File: setup.py (hunk 2 above)
Comment (nephomaniac) [reply setup.py:5]: @lincolnthomas  leaving level 7 for you ;-)

**Pull request merged**